src/lib/MaskedInput.tsx

```diff
@@ -181,7 +183,7 @@ export default class MaskedInput extends Component<MaskedInputProps> {
     e.preventDefault();
     this._updateMaskSelection();
     // getData value needed for IE also works in FF & Chrome
-    if (this.mask.paste(e.clipboardData.getData('Text'))) {
+    if (this.mask.paste(e.clipboardData.getData('Text').replace(/[^0-9a-z]/gi, ''))) {
```

Comment:
Why replace the first character?

Reply:
My thought on this is that we should remove any unwritable character from pasted content since the source may have a different format. For example, I may use (xxx) xxx-xxx as the phone number format but the user may copy and paste a number from another app that uses xxx-xxx-xxxx as the format. Without stripping out the special characters, pasting from that app will likely not work properly.

What do you think? Thx